source/components/status-label/status-label.less

```diff
@@ -0,0 +1,56 @@
+hx-status {
+  background-color: transparent;
+  border: 1px solid;
+  border-color: @gray-750;
+  border-radius: 1em; // 10px
+  color: @gray-750;
+  display: inline-block;
+  font-size: 0.625rem; // 10px
+  line-height: 1em;
+  max-width: 100%;
+  min-width: 120px;
+  padding: 0.2em 0.8em; // 2px 8px
+  text-align: center;
+  text-transform: uppercase;
+  width: inherit;
```

Comment:
What problem is this property solving? This seems strange to inherit width.

Reply:
I added it in a certain situation, but can't recreate it now. I'm just removing it for now.